src/mumble/Plugins.cpp

```diff
@@ -669,8 +660,8 @@ void Plugins::fetchedUpdatePAPlugins(QByteArray data, QUrl) {
 		}
 	}
 
-	if (qd != qdu) {
-		libs = qdu.entryInfoList();
+	if (systemDir != downloadDir) {
+		libs = downloadDir.entryInfoList();
```

Comment:
The following code seems to be pretty much the same as in the first iteration. Therefore the code should be outsourced into a function so that it can be reused for all use-cases :point_up:

Reply:
I agree.

Should I do it in this PR?

Reply:
Yeah, why not ^^
Just make it a separate commit :)